src/components/Output.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] [ OPTIONAL]  KIndly fix linter issue manually after removing these lines. This is going mandatory requirement in further process.